addons/account/models/account_journal.py

```diff
@@ -615,7 +624,7 @@ def _get_journal_bank_account_balance(self, domain=None):
         self.ensure_one()
         self.env['account.move.line'].check_access_rights('read')
 
-        accounts = self.default_debit_account_id + self.default_credit_account_id
+        accounts = self.default_account_id
```

Comment:
Since there is only one account left, clean the code plz. For example `('account_id', 'in', tuple(accounts.ids))` is useless, to replace by `('account_id', '=', self.default_account_id.id)`